src/components/vm/consoles/vnc.jsx

```diff
@@ -68,6 +69,18 @@ const Enum = {
     KEY_DELETE: 111,
 };
 
+export class VncState extends ConsoleState {
+    constructor() {
+        super();
+        this.sizeMode = "none";
+    }
+
+    setSizeMode(val) {
+        this.sizeMode = val;
+        this.update();
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2015-3fe0b058-20250716-111222-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)